docs/user-guide/installation.md

```diff
@@ -39,24 +40,29 @@ fireactions-server-1 ansible_host=<IP_ADDRESS> ansible_user=<SSH_USER>
         github:
           app_id: <APP_ID>
           app_private_key: |
-            <APP_PRIVATE_KEY>
+            -----BEGIN RSA PRIVATE KEY-----
+            <PRIVATE_KEY_HERE>
+            -----END RSA PRIVATE KEY-----
         debug: true
         pools:
-        - name: example
-          max_runners: 10
-          min_runners: 1
+        - name: myPool
+          max_runners: 100
+          min_runners: 10
           runner:
-            name: example
-            image: <IMAGE>:<IMAGE_TAG>
+            name: myRunnerGroup
+            # ref: https://github.com/hostinger/fireactions-images
+            image: ghcr.io/hostinger/fireactions-images/ubuntu20.04:latest
             image_pull_policy: IfNotPresent
             group_id: 1
-            organization: hostinger
+            organization: myorganization
             labels:
             - self-hosted
             - fireactions
           firecracker:
             binary_path: firecracker
-            kernel_image_path: /usr/local/share/firecracker/vmlinux.bin
+            # obtained from https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.10/x86_64/vmlinux-5.10.223
```

Comment:
I think this will need separate documentation page, as we also create our own kernel images [here](https://github.com/hostinger/fireactions-images/tree/main/kernels). Need to define the differences, I'll handle this upcoming week, then You can update the reference here.

It should work with the one from Firecracker, even though some crucial options will be missing.

Reply:
Hi, sorry for the wait, been busy. You can update it to [this](https://fireactions.io/user-guide/kernels/) reference, change the PR title to match conventional commit guidelines (we have squash merge with title as commit message enabled) and it's good to go!

Regarding the differences between kernels - I really can't remember, should have documented when building:( But what I can remember is that it's built off the Firecracker kernel config. I think we had some issues with running Docker/Kubernetes workloads.